README.md

```diff
@@ -53,13 +53,23 @@ There are two ways you can use this module:
 
 1. You can install it through the `deno` command.
     ```sh
-    $ deno install --allow-net='cdn.deno.land,api.deno.land,x.nest.land' --allow-read='.' --allow-write='deps.ts' https://deno.land/x/dmm@v1.1.5/mod.ts
+    $ deno install \
+      --allow-net='cdn.deno.land,api.deno.land,x.nest.land,raw.githubusercontent.com' \
```

Comment:
breaking change? anything currently using dmm will break and will need t o be fixed right?

Reply:
is this a breaking change because i added raw.githubusercontent.com ?

Reply:
yeah, i guess cause we are pulling from that new url now, though it might only bee breaking for some people, because only some might do `--allow-net='cdn.deno.land,...'` and if they do, dmm will break cause it wont be allowed to fetch from rawgithubuserconteent

Reply:
ohhhhhh. ok. i see. i added it because when i did `dmm` on dmm itself, it was trying to fetch raw.githubusercontent.com stuff because dmm (in this branch) uses raw.githubusercontent.com for `BumperService`, `CliService` and `LoggerService`

Reply:
we can remove raw.githubusercontent.com from this

Reply:
i mean i'm cool with it being a breaking change, just thought this means we should go to v2? idk how 'breaking' this is or if it's classed as a major version increment

Reply:
i didn't think it was a breaking change because we're adding to the `--allow-net` list instead of taking away from the list

Reply:
Oh wait maybe it isn't, cause when a user upgrades dmm they'll see they'll need to add the new url to the net access in the release, so i guess it isn't really a problem.. ignore me then